index.js

```diff
@@ -58,16 +78,18 @@ module.exports = function (fastify, opts, next) {
           })
         }
       })
-    }, (err, stats) => {
+    }, (err, files) => {
       if (err) {
         next(err)
         return
       }
 
+      const stats = [].concat(...files)
```

Comment:
Probably I'm missing something, what's the need of this line?

Reply:
`files` can contain further arrays if there are multiple files in a folder:

```js
[ { skip: true,
    file: '/api/services/.DS_Store' },
  [ { skip: false,
      opts: [Object],
      file: '/api/services/orders/get.js' },
    { skip: false,
      opts: [Object],
      file: '/api/services/orders/list.js' } ] ]
```

`[].concat(...files)` flattens the array.